src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -467,6 +466,32 @@ def solver_options(self):
         return options
 
 
+class EndScenarioSizingStaged(EndScenarioSizingHIGHS):
```

Comment:
Now the EndscenarioSizingStaged is always done with Highs as gurobi will also be implemented in MVP, don't we want to have this class inheriting from EndScenarioSizing, and then make one additional class EndScenarioSizingStagedHIGHS which inherits this class and only changes the solver options? Then both solvers Gurobi and Highs can be used.